torchaudio/backend/utils.py

```diff
@@ -33,6 +35,16 @@ def list_audio_backends() -> List[str]:
     return backends
 
 
+_USE_SOUNDFILE_LEGACY_INTERFACE = True
+
+
+def use_soundfile_legacy_interface(value: bool):
+    """Switch soundfile backend interface.
+    """
+    global _USE_SOUNDFILE_LEGACY_INTERFACE
```

Comment:
I'm not even sure you need to use the ```global``` keyword here

Reply:
You might also want a getter and a setter to help send warnings when this variable is being changed a lot